boards/arm/b_l072z_lrwan1/b_l072z_lrwan1.dts

```diff
@@ -73,6 +74,26 @@ arduino_i2c: &i2c1 {};
 
 &spi1 {
 	status = "okay";
+	cs-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
+
+	lora: sx1276@0 {
+		compatible = "semtech,sx1276";
+		reg = <0>;
+		label = "sx1276";
+		reset-gpios = <&gpioc 0 GPIO_ACTIVE_LOW>;
+		dio-gpios = <&gpiob 4 GPIO_ACTIVE_HIGH>,
+			<&gpiob 1 GPIO_ACTIVE_HIGH>,
+			<&gpiob 0 GPIO_ACTIVE_HIGH>,
+			<&gpioc 13 GPIO_ACTIVE_HIGH>,
+			<&gpioa 5 GPIO_ACTIVE_HIGH>,
+			<&gpioa 4 GPIO_ACTIVE_HIGH>;
```

Comment:
`dio-gpios` are all input. I think those should have PULL DOWN :)

Reply:
Ugh, I was a bit too ambitious there. :(